### ISSUE TITLE
Rultor on Bitbucket issue system

### ISSUE THREAD

**Issue by tmoreira2020** [yegor256/rultor#912]
Is there any plans to integrate Rultor with bitbucket issue tracker? More info at https://confluence.atlassian.com/display/BITBUCKET/Use+the+issue+tracker

**Comment by yegor256** [comment]
@tmoreira2020 yes, of course, we're planning to do it. maybe you're interested to contribute? :)

**Comment by alex-palevsky** [comment]
@tmoreira2020 I added milestone `2.0` to this issue, let me know if there has to be something else

**Comment by alex-palevsky** [comment]
@yegor256 it is yours,help us

**Comment by tmoreira2020** [comment]
@yegor256 Right now it is tough for me but I could give a try in a couple of months.

**Comment by yegor256** [comment]
@tmoreira2020 thanks! maybe we'll solve it earlier, but I doubt...

**Comment by yegor256** [comment]
@alex-palevsky assign someone else here pls

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else here pls

@yegor256 -30 points to your rating

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else here pls

@yegor256 no problem, I will try to find somebody else

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear I will try to find someone else
